packages/core/src/mesh/PrimitiveMesh.ts

```diff
@@ -1,40 +1,39 @@
 import { Engine } from "../Engine";
-import { Buffer } from "../graphic/Buffer";
-import { BufferBindFlag } from "../graphic/enums/BufferBindFlag";
-import { BufferUsage } from "../graphic/enums/BufferUsage";
-import { IndexFormat } from "../graphic/enums/IndexFormat";
-import { VertexElementFormat } from "../graphic/enums/VertexElementFormat";
 import { Mesh } from "../graphic/Mesh";
-import { VertexElement } from "../graphic/VertexElement";
 import { GLCapabilityType } from "../base/Constant";
+import { ModelMesh } from "./ModelMesh";
+import { Vector2, Vector3 } from "@oasis-engine/math";
 
 /**
  * Used to generate common primitve meshes.
  */
 export class PrimitiveMesh {
   /** The max number of indices that Uint16Array can support. */
-  private static _uint16VertexLimit: number = 65535;
+  private static _Uint16VertexLimit: number = 65535;
 
   /**
    * Create a sphere mesh.
    * @param engine - Engine
    * @param radius - Sphere radius
    * @param segments - Number of segments
+   * @param noLongerAccessible - No longer access the vertices of the mesh after creation
    * @returns Sphere mesh
```

Comment:
I found you write ```@returns Plane modle mesh``` in ```createSphere```, other ```createXX()``` write like this is better

Reply:
+1